.circleci/config.yml

```diff
@@ -149,6 +149,7 @@ jobs:
       - run: TELEPRESENCE_VERSION=$CIRCLE_TAG make push-images push-executable
       - run: |
           [[ $CIRCLE_TAG == *-* ]] || TELEPRESENCE_VERSION=$CIRCLE_TAG make promote-to-stable
+          [[ $CIRCLE_TAG == *-nightly-* ]] && TELEPRESENCE_VERSION=$CIRCLE_TAG make promote-nightly
```

Comment:
I have mixed feelings about creating actual Git tags, instead of just putting the shorthash in the docker tag.

Reply:
Using `&&` chains make me nervous.  Because `set -e` is weird, they won't cause it to bail, but if the condition doesn't trigger it will set `$?` to non-zero, which I think will cause CircleCI to consider the step to have failed.

So I guess change this to `[[ $CIRCLE_TAG != *-nightly-* ]] ||`.

Reply:
I have removed the tagging and put building + pushing + promoting of artifacts in the "publish-*-nightly" jobs for linux + macOS.